relay-profiling/src/android/chunk.rs

```diff
@@ -40,7 +39,6 @@ pub struct Metadata {
 
     #[serde(default)]
     duration_ns: u64,
-    #[serde(default)]
```

Comment:
With this change, won't all profiles from old SDKs fail to parse?

Reply:
We haven't released any Android `SDK` with continuous profiling support yet.

 Older SDKs are supporting the "legacy" Android profiles.